x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml

```diff
@@ -30,7 +30,7 @@ setup:
         - method: POST
           path: /_query
           parameters: []
-          capabilities: [ snapshot_test_for_telemetry ]
+          capabilities: [ snapshot_test_for_telemetry, fn_bit_length ]
```

Comment:
My first idea would have been to use only `fn_bit_length`. Gut-feeling wise, that should be the only one needed, but I am not certain and I would try it like this as well and maybe dig a bit deeper to understand exactly why XPackRestIT  fails.

Reply:
Adjusted with [Only use most recent capability in usage tests](https://github.com/elastic/elasticsearch/pull/115792/commits/762be473240a74df1ae5c4b55d74e8e9af2e9dfa). Let's see ... 👀 If it still fails I'll try to dig in to see what's going wrong

Reply:
Just to clarify my own understanding of bwc rest compability tests...How does this exactly work? Does the old usage test (from branch `8.x...`; basically mimicking an "old" client) gets executed against current `main`? If so, this would make sense as `main` has now one ES|QL function more. Also adding the `fn_bit_length` capability wouldn't solve the problem, because this change is not present on the older branch, right?

Reply:
> Does the old usage test (from branch 8.x...; basically mimicking an "old" client) gets executed against current main?

Yes. This is documented [here](https://github.com/elastic/elasticsearch/blob/main/REST_API_COMPATIBILITY.md#testing): `"The build system will download the latest prior version of the YAML rest tests and execute them against the current cluster version."`. 
(I thought it would work as the other BWC tests, that run new queries against old code, but it's the other way around. TIL.)
So indeed, updating the test on main (9.x) has no effect on the test run: we need to update the 8.x test.
@astefan, I think we might need to either [skip these tests](https://github.com/elastic/elasticsearch/blob/43aa301688892890e2cf8c3f783b7cc190561415/x-pack/plugin/build.gradle#L205) in REST BWC tests on main, or update the way we check for the functions count on 8.x (which might be tricky, but probably checking against a lower limit _might_ be safe).

Reply:
Also, this s/`snapshot_test_for_telemetry`/`fn_bit_length` is probably then no longer necessary.